sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpSender.cs

```diff
@@ -361,106 +361,105 @@ public override async Task CloseAsync(CancellationToken cancellationToken)
         /// <summary>
         ///
         /// </summary>
-        /// <param name="message"></param>
+        /// <param name="messages"></param>
         /// <param name="cancellationToken"></param>
         /// <returns></returns>
```

Comment:
Are these doc comments supposed to be empty?

Reply:
Nope we have a work item to clean up the documentation for all of our internal members.